app/controllers/api/api_controller.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+module Api
+  class ApiController < ApplicationController
```

Comment:
This file should be deleted since it's not of use for now. Always think small steps!